CougSat1-Ground/src/space/cougs/ground/packetprocessing/PacketHeader.java

```diff
@@ -0,0 +1,82 @@
+package space.cougs.ground.packetprocessing;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+import space.cougs.ground.packetprocessing.downlinkpackets.DownlinkPacket;
+import space.cougs.ground.packetprocessing.downlinkpackets.Payload1Configuration;
+import space.cougs.ground.packetprocessing.downlinkpackets.Payload2Configuration;
+import space.cougs.ground.packetprocessing.downlinkpackets.Payload3Configuration;
+import space.cougs.ground.packetprocessing.downlinkpackets.RadioConfiguration;
+import space.cougs.ground.packetprocessing.downlinkpackets.Telemetry;
+import space.cougs.ground.satellites.CougSat;
+
+public class PacketHeader {
+
+	private final List<CougSat> satellites = new ArrayList<CougSat>();
+	private DownlinkPacket currentPacket;
+
+	public PacketHeader() {
+
+	}
+
+	public boolean decodePacketSwitcher(String filePath) {
+
+		File file = new File(filePath);
+		int firstByte = -1;
+		try {
+
+			FileInputStream inFile = new FileInputStream(file);
+
+			firstByte = inFile.read();
+
+			inFile.close();
+
+		} catch (IOException e) {
+			// TODO Auto-generated catch block
+			e.printStackTrace();
+		}
+		if (firstByte == -1) {
+			System.out.printf("Error 4 - Incorrect filePath for %s", filePath);
+			return false;
+		}
+
+		switch (firstByte & 0x1F) {
```

Comment:
Magic numbers are bad, make into a private static final int. Include word MASK. This happens below too. Make it the bitwise NOT of this one